test/new-e2e/pkg/runner/ci_profile.go

```diff
@@ -74,7 +88,7 @@ func NewCIProfile() (Profile, error) {
 
 	return ciProfile{
 		baseProfile: newProfile("e2eci", ciEnvironments, store, &secretStore, outputRoot),
-		ciUniqueID:  "ci-" + jobID + "-" + projectID,
+		ciUniqueID:  "ci-" + uniqueID + "-" + projectID,
```

Comment:
As someone might not know the diff between at job ID and a pipeline ID just by looking at it, shouldn't we add a `ci-init-*`